docs/guides/index.md

```diff
@@ -138,11 +141,11 @@ some motion into the scene:
 To interact with the box via clicking or gazing, we can use a cursor which we
 place as a child of the camera such that it is fixed to the screen. When we
 don't define a camera, the scene will inject a default camera, but in this case
-to add a cursor, we will need to define one.
+to add a cursor, we will need to explicitly define a camera.
 
-Then what we can tell the animation only to start when the cursor clicks the
-box event, using the animation's `begin` attribute. The cursor will emit the
-`click` event on the box, and the animation will listen to it:
+Next we configure the animation to start when the cursor clicks the
```

Comment:
seems we should use prose when we can, `tell the animation` vs `configure the animation`